CHANGELOG.md

```diff
@@ -2,8 +2,12 @@
 
 ## Version 2.3.0
 
+- Added Prettier code style
+- Deleted .npmignore and switched to `package.json` files field
+- Added [CONTRIBUTING](./CONTRIBUTING.md) and [CODE_OF_CONDUCT](./CODE_OF_CONDUCT.md) docs
 - Switched to `moo-color` for color parsing
```

Comment:
why switch to a new project?

Reply:
`moo-color` supports the `"transparent"` color.

Reply:
`moo-color` is written by  contributor? I'm worried about compatibility.

Maybe the best way is to send pr to `parse-color` for support.

Reply:
I checked the source of `moo-color`. It's very well done. I think it's okay for compatibility.  Doesn't travis already test node 8?

Reply:
Also, `parse-color` hasn't been updated in 8 years, but `moo-color` still has active support. I think this is a non-issue.

Reply:
ok~